spec/integration/admin_view_spec.rb

```diff
@@ -1,37 +1,32 @@
 require "spec_helper"
 
 describe "the admin view", type: :feature do
-  # include Capybara::DSL
+  include Capybara::DSL
 
-  before(:each) do
-    user = User.find_by(:role, 1)
-    mock(ApplicationController).stub!(:current_user).and_return(user)
-  end
+  #before(:each) do
+    #user = User.find_by(role: 1)
+    #mock(ApplicationController).stub!(:current_user).and_return(user)
+    #visit admin_path
```

Comment:
Incorrect indentation detected (column 4 instead of 2).<br>Missing space after #.